src/box.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {BoxProps} from './types/box-types'
+import {propTypes} from './enhancers'
+import enhanceProps from './enhance-props'
+
+export default class Box extends React.Component<BoxProps, {}> {
+  static displayName = 'Box'
+
+  static propTypes = {
+    ...propTypes,
+    innerRef: PropTypes.func,
+    is: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+    boxSizing: propTypes.boxSizing
+  }
+
+  static defaultProps = {
+    innerRef: null,
+    is: 'div',
+    boxSizing: 'border-box'
+  }
+
+  render() {
+    const {is = 'div', innerRef, children, ...props} = this.props
```

Comment:
Do we still need  is `=div` since we already have defaultProps?

Reply:
Technically, no, but I think some versions of TypeScript will complain because they don't understand `defaultProps`